packages/ckeditor5-list/tests/listproperties/ui/listpropertiesview.js

```diff
@@ -708,6 +708,17 @@ describe( 'ListPropertiesView', () => {
 				sinon.assert.notCalled( spy );
 				expect( view.startIndexFieldView.errorText ).to.equal( 'Start index must be greater than 0.' );
 			} );
+
+			it( 'should not fire #listStart upon #input but display an error if the numeric value is NaN', () => {
```

Comment:
There are still no tests or assertions to verify the disappearance of error messages after typing the correct value.

Reply:
okay, but I don't see them in all other tests, anyway good idea